### PR TITLE
feat: add command execution track and shebang

### PR DESCRIPTION
**Pull request by idbeta**
add command execution track and shebang

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -1,12 +1,16 @@
+#!/bin/bash
+
+set -ex
+
 version=1.5
 # version=master
 iteration=0
 
-sudo luarocks remove apisix --local
+sudo luarocks remove apisix --local || true
```
Comment (membphis) [inline run.sh:9]: this is wrong. we need to remove the apisix in folder `/tmp/apisix/usr/local/apisix/deps`

it should be a bug for a long time

File: run.sh [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 sudo rm -rf /usr/local/apisix/
 rm -rf /tmp/apisix
 
-wget https://github.com/apache/apisix/raw/master/rockspec/apisix-$version-$iteration.rockspec
+wget https://github.com/apache/apisix/raw/master/rockspec/apisix-$version-$iteration.rockspec -O apisix-master-0.rockspec
```
Comment (membphis) [inline run.sh:13]: `apisix-master-0.rockspec` is wrong

**Review by membphis — CHANGES_REQUESTED** [review]

File: run.sh (hunk 1 above)
Comment (membphis) [reply run.sh:9]: I think we should use `luarocks purge --tree=/tmp/apisix/usr/local/apisix/deps` here

Comment (membphis) [reply run.sh:9]: or we can clean the folder `/tmp/apisix/usr/local/apisix/deps`, this is a simpler way

Comment (idbeta) [reply run.sh]: fixed.

File: run.sh (hunk 2 above)
Comment (idbeta) [reply run.sh]: fixed.

**Review by membphis — APPROVED** [review]

**Pull request merged**